spec/models/order_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Order, type: :model do
   context '#add_item' do
-    it 'adds an item' do
+    xit 'adds an item' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.